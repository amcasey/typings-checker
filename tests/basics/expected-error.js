// This file contains an error which is expected to be caught.
"use strict";
var x = { a: 'a', b: 2 };
// $ExpectError Property 'c' does not exist
x.c;
Object.defineProperty(exports, "__esModule", { value: true });
exports.default = null;
