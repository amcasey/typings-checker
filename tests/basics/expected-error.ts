// This file contains an error which is expected to be caught.

const x = {a: 'a', b: 2};
// $ExpectError Property 'c' does not exist
x.c
