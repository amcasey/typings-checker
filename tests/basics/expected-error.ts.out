tests/basics/expected-error.ts: 1 / 1 checks passed.
